react/features/base/devices/middleware.js

```diff
@@ -1,9 +1,11 @@
 /* global APP */
 
-import UIEvents from '../../../../service/UI/UIEvents';
-
+import { CONFERENCE_JOINED } from '../conference';
+import { processExternalDeviceRequest } from '../../device-selection';
```

Comment:
style: sort these as usual

Reply:
What do you mean? I think they are already sorted as usual, aren't they? 

Just to make sure, the "usual" is alphabetical sorting on filenames by ignoring the dots and slashes, right?

Reply:
../../foo goes before ../bar

Reply:
and an empty line between the 2

Reply:
We agreed with @saghul that we will leave it as it is until we introduce a rule for sorting the imports.